internal/readme/stats.go

```diff
@@ -99,7 +99,7 @@ func statsExamples() {
 	_ = v
 
 	// START unregisterView
-	if v.Unregister(); err != nil {
+	if stats.UnregisterView(v); err != nil {
```

Comment:
Can you also run embedmd to re-generate the root README file?

Reply:
Ah yes, thanks for that catch. Let me do so.

Reply:
@songy23 done, PTAL.